### PR TITLE
style: fix checkstyle findings

### PR DESCRIPTION
**Pull request by jdrueckert**
* remove unused import
* fix LineLengthChecks

### CODE REVIEW

**Comment by jdrueckert** [comment]
[Fix Checkstyle Findings](https://trello.com/c/SvpELrak/121-fix-checkstyle-findings)

File: engine/src/main/java/org/terasology/engine/monitoring/impl/PerformanceMonitorImpl.java [hunk 1]
```diff
@@ -113,8 +113,12 @@ public Activity startActivity(String activityName) {
...
         if (!activityStack.isEmpty()) {
             ActivityInfo currentActivity = activityStack.peek();
-            currentActivity.ownTime += newActivity.startTime - ((currentActivity.resumeTime > 0) ? currentActivity.resumeTime : currentActivity.startTime);
-            currentActivity.ownMem += (currentActivity.startMem - newActivity.startMem > 0) ? currentActivity.startMem - newActivity.startMem : 0;
+            currentActivity.ownTime += newActivity.startTime - ((currentActivity.resumeTime > 0)
+                    ? currentActivity.resumeTime
+                    : currentActivity.startTime);
+            currentActivity.ownMem += (currentActivity.startMem - newActivity.startMem > 0)
+                    ? currentActivity.startMem - newActivity.startMem
+                    : 0;
```
Comment (skaldarnar) [inline engine/src/main/java/org/terasology/engine/monitoring/impl/PerformanceMonitorImpl.java:121]: Is it just me who finds this hard to read and comprehend? Have the conditional outside is already hard, but having it as part of the formula .... :see_no_evil:

**Review by skaldarnar — APPROVED** [review]
There's so much code in there I don't want to look at :see_no_evil: Not sure whether I like the positions of all the line breaks, but bringing the warnings down is a good step :+1:

File: engine/src/main/java/org/terasology/engine/world/generation/facets/base/BaseFacet3D.java [hunk 2]
```diff
@@ -32,13 +32,15 @@ protected final int getRelativeIndex(int x, int y, int z) {
         if (!relativeRegion.contains(x, y, z)) {
             throw new IllegalArgumentException(String.format("Out of bounds: (%d, %d, %d) for region %s", x, y, z, relativeRegion.toString()));
         }
-        return x - relativeRegion.minX() + relativeRegion.getSizeX() * (y - relativeRegion.minY() + relativeRegion.getSizeY() * (z - relativeRegion.minZ()));
+        return x - relativeRegion.minX() + relativeRegion.getSizeX()
+                * (y - relativeRegion.minY() + relativeRegion.getSizeY() * (z - relativeRegion.minZ()));
```
Comment (skaldarnar) [inline engine/src/main/java/org/terasology/engine/world/generation/facets/base/BaseFacet3D.java:36]: Does this fit into one line?
```suggestion
        return x - relativeRegion.minX() 
                + relativeRegion.getSizeX() * (y - relativeRegion.minY() + relativeRegion.getSizeY() * (z - relativeRegion.minZ()));
```

**Review by skaldarnar — APPROVED** [review]

**Pull request merged**